Makefile

```
@@ -308,11 +308,21 @@ define find_all_clojure_files
 $$(comm -23 <(sort <(git ls-files --cached --others --exclude-standard)) <(sort <(git ls-files --deleted)) | grep -e \.clj$$ -e \.cljs$$ -e \.cljc$$ -e \.edn)
 endef
 
+# Lint Clojure files with clj-kondo.
+#
+# Args:
+#   $1: When equal to true, print warnings.
+define lint_clojure_files
+	$(if $(filter true,$1), \
+			clj-kondo --config .clj-kondo/config.edn --cache false --fail-level error --lint src, \
+			clj-kondo --config .clj-kondo/config.edn --cache false --fail-level error --lint src | grep -v ': warning: ')
+endef
+
 lint: export TARGET := clojure
```

If you are adding a parameter to this target you should indicate that in the target:
```suggestion
lint: export TARGET := clojure
lint: export CLJ_LINTER_PRINT_WARNINGS ?= false
```

---

Done https://github.com/status-im/status-mobile/pull/17491/commits/2f2f4dc624995df3ad191bd8eb3ade054c569a4c